dakota/oat/cooling-time/README.md

```diff
@@ -0,0 +1,24 @@
+# Hello. Welcome to the One-at-a-time Cooling time sensitivity analysis study. 
+To run the simulation, type this into the command line: 
```

Comment:
More detail here, as intro. Give a sentence answering the question: what's the point of this folder?